homeassistant/components/plex/__init__.py

```diff
@@ -138,6 +141,10 @@ def _setup_plex(hass, config):
 
     entry.add_update_listener(async_options_updated)
 
+    hass.data[PLEX_DOMAIN][REFRESH_LISTENERS][server_id] = async_track_time_interval(
```

Comment:
We need to change how we unsubscribe this when unloading the entry. It's currently run in the executor.